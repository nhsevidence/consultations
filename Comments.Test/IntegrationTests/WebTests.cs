using Comments.Test.Infrastructure;
using System.Threading.Tasks;
using Xunit;

namespace Comments.Test.IntegrationTests
{
    public class WebTests : TestBase
    {
        [Fact]
        public async Task Get_Consultations_Homepage()
        {
            // Act
            var response = await _client.GetAsync("/consultations"); //this route is temporary.
            response.EnsureSuccessStatusCode();

            var responseString = await response.Content.ReadAsStringAsync();

            // Assert
            responseString.ShouldMatchApproved();
        }

        [Fact]
        public async Task Get_React_Document_Page()
        {
            // Act
            var response = await _client.GetAsync("/consultations/1/1/introduction");
            response.EnsureSuccessStatusCode();

            var responseString = await response.Content.ReadAsStringAsync();

            // Assert
            responseString.ShouldMatchApproved();
        }
    }
}
