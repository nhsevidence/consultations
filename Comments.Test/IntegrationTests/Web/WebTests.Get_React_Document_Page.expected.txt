
<!DOCTYPE html>
  <html class="no-js" lang="en-GB">
    <head>
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="X-UA-Compatible" content="IE=edge,chrome=1">
          <meta charset="utf-8">
            <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
              <title data-react-helmet="true">
              </title>
              <base href="/consultations/"/>
                <meta name="theme-color" content="#004650">
                  <script src="/consultations/vendor/modernizr-custom.js">
                  </script>
                  <link rel="manifest" href="/consultations/manifest.json">
                    <meta name="msapplication-TileColor" content="#f6f6f6"/>
                    <meta name="msapplication-TileImage" content="//cdn.nice.org.uk/V3/Content/nice/favicon-144.png"/>
                      <link rel="shortcut icon" href="//cdn.nice.org.uk/V3/Content/nice/favicon.ico"/>
                        <link rel="apple-touch-icon-precomposed" href="//cdn.nice.org.uk/V3/Content/nice/favicon-152.png">
                          <link rel="apple-touch-icon" href="//cdn.nice.org.uk/V3/Content/nice/favicon-152.png">
                            <link rel="apple-touch-icon-precomposed" sizes="120x120" href="//cdn.nice.org.uk/V3/Content/nice/favicon-120.png">
                              <link rel="apple-touch-icon" sizes="120x120" href="//cdn.nice.org.uk/V3/Content/nice/favicon-120.png">
                                <link rel="apple-touch-icon-precomposed" sizes="152x152" href="//cdn.nice.org.uk/V3/Content/nice/favicon-152.png">
                                  <link rel="apple-touch-icon" sizes="152x152" href="//cdn.nice.org.uk/V3/Content/nice/favicon-152.png">
                                    <link href="//fonts.googleapis.com/css?family=Lato:400,700,900" rel="stylesheet"/>
                                      <link rel="stylesheet" href="/consultations/main.css">
                                        <!--[if lt IE 9]>
                                          <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js">
                                          </script>
                                          <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.7/es5-shim.min.js">
                                          </script>
                                          <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.10/es5-sham.min.js">
                                          </script>
                                          <script src="//cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.min.js">
                                          </script>
                                          <![endif]-->
                                          </head>
                                          <body>
                                            <div id="root">
                                              <div class="Drawer   ">
                                                <div class="Drawer__controls">
                                                  <button class="Drawer__toggleOpen">
                                                    <span class="icon icon--chevron-left" aria-hidden="true">
                                                    </span>
                                                  </button>
                                                </div>
                                                <div class="Drawer__main">
                                                  <div class="Drawer__mainControls">
                                                    <img style="opacity:0.3" src="[object Object]" alt="Icon Dock Small" width="40"/>
                                                    <img style="opacity:1" src="[object Object]" alt="Icon Dock Extended" width="40"/>
                                                  </div>
                                                </div>
                                              </div>
                                              <h1>
                                                Loading...
                                              </h1>
                                            </div>
                                            <script src="//cdn.nice.org.uk/V3.1/Scripts/nice/NICE.TopHat.min.js" data-environment="Beta" data-search="/search?q=%term" data-typeaheadtype="remote" data-typeaheadsource="/autocomplete?%query" data-search-placeholder="Search NICE..." data-cookie="true" async>
                                            </script>
                                            <script>
                                              window.__PRELOADED__={};
                                            </script>
                                            <script type="text/javascript" src="/consultations/static/js/main.js">
                                            </script>
                                          </body>
                                        </html>